scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/JobData.java

```diff
@@ -701,7 +701,10 @@ JobUsage toJobUsage() {
                                          getProjectName(),
                                          jobId.value(),
                                          getJobName(),
-                                         getFinishedTime() - getStartTime());
+                                         getFinishedTime() - getStartTime(),
+                                         getStatus().toString(),
```

Comment:
no need for toString here

Reply:
We do need `toString()` because constructor requires String while `getStatus()` returns JobStatus.